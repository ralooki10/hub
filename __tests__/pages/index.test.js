import renderer from 'react-test-renderer'
import Page from '../../pages/index.js'

describe('homepage look and feel', () => {
    it('should render correctly', () => {
        const component = renderer.create(<Page />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
});

