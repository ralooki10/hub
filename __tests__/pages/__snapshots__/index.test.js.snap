// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`homepage look and feel should render correctly 1`] = `
<div>
  <p>
    Super bare page I'm not sure what to do with
  </p>
</div>
`;
